lnwallet/channel.go

```diff
@@ -1904,6 +2014,34 @@ func (lc *LightningChannel) restorePendingRemoteUpdates(
 	return nil
 }
 
+// restorePeerLocalUpdates restores the acked local log updates the peer still
+// needs to sign for.
+func (lc *LightningChannel) restorePeerLocalUpdates(updates []channeldb.LogUpdate,
+	remoteCommitmentHeight uint64) error {
+
+	lc.log.Debugf("Restoring %v local updates that the peer should sign",
+		len(updates))
+
+	for _, logUpdate := range updates {
+		logUpdate := logUpdate
+
+		payDesc, err := lc.localLogUpdateToPayDesc(
+			&logUpdate, lc.remoteUpdateLog, remoteCommitmentHeight,
+		)
+		if err != nil {
+			return err
+		}
+
+		lc.localUpdateLog.restoreUpdate(payDesc)
+
+		if payDesc.EntryType != FeeUpdate {
```

Comment:
Could add a comment here that only adds are stored under this key, so we only need this special case for fee updates.

Reply:
PTAL at comment